File: PHP/library/DataModelsRepoSql08/CustomerOrderRepo.php
<?php

namespace DataModelsRepoSql08;

require_once ('library/IDataModelsRepo/ICustomerOrderRepo.php');

use IDataModelsRepo\ICustomerOrderRepo;
use DataModels\ItemDetails;
use DataModels\CustomerInfo;
use DataModels\CustomerOrder;

class CustomerOrderRepo implements ICustomerOrderRepo {
	public $connStr;
	
	function __construct($connString) {
		$connStr = $connString;
	}
	public function GetOrderById($id) {
	    $c = new CustomerOrder();
		$c->orderId = 1;
		$c->orderDate = "2012-07-08";
		return $c;
	}
	
	
	function __destruct() {
	}
}

?>

File: CSharp/FailSafeTDD_Example/src/app/DataModelsRepoSql08/CustomerOrderRepo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DataModels;
using IDataModelsRepo;

namespace DataModelsRepoSql08
{
    public class CustomerOrderRepo: ICustomerOrderRepo
    {
        public string ConnStr { get; set; }

        public CustomerOrderRepo(string connStr)
        {
            ConnStr = connStr;
        }
        public CustomerOrder GetOrderById(int id)
        {
            throw new NotImplementedException();
        }
    }
}
